lib/elixir/lib/list.ex

```diff
@@ -1,8 +1,8 @@
 defmodule List do
   @moduledoc """
   Implements functions that only make sense for lists
```

Comment:
Let's shorten this one too. Maybe: "Specialized functions that works only on lists". The rest can be its own paragraph.